lms/resources/_js_config.py

```diff
@@ -200,7 +243,7 @@ def _add_canvas_speedgrader_settings(self, **kwargs):
                 "lis_outcome_service_url": lis_outcome_service_url,
                 "learner_canvas_user_id": self._request.params["custom_canvas_user_id"],
                 **kwargs,
-            },
+            }
```

Comment:
```suggestion
            },
```

Reply:
I formatted these files with `make format`. Does that not deal with this as expected?

Reply:
I think I may have had an older version of black installed globally and used by my editor, although that doesn't explain why `make format` didn't correct it if so.